python/tests/test_dwelling.py

```diff
@@ -10,30 +10,52 @@
 @pytest.fixture
 def sample_input_d() -> reader.InputData:
     return {
-        'EVAL_ID': 123,
+        'EVAL_ID': '123',
         'EVAL_TYPE': 'D',
         'ENTRYDATE': '2018-01-01',
         'CREATIONDATE': '2018-01-08 09:00:00',
         'MODIFICATIONDATE': '2018-06-01 09:00:00',
         'CLIENTCITY': 'Ottawa',
         'forwardSortationArea': 'K1P',
         'HOUSEREGION': 'Ontario',
-        'YEARBUILT': 2000,
+        'YEARBUILT': '2000',
+        'ceilings': [
+            {
+                'label': 'Main attic',
+                'type_english': 'Attic/gable',
+                'type_french': 'Combles/pignon',
+                'nominal_rsi': '2.864',
+                'effective_rsi': '2.9463',
+                'area': '46.4515',
+                'length': '23.875',
+            }
+        ]
     }
 
 
 @pytest.fixture
 def sample_input_e() -> reader.InputData:
     return {
-        'EVAL_ID': 123,
+        'EVAL_ID': '123',
         'EVAL_TYPE': 'E',
         'ENTRYDATE': '2018-02-01',
         'CREATIONDATE': '2018-02-08 09:00:00',
         'MODIFICATIONDATE': '2018-06-01 09:00:00',
         'CLIENTCITY': 'Montreal',
         'forwardSortationArea': 'G1A',
         'HOUSEREGION': 'Quebec',
-        'YEARBUILT': 2001,
+        'YEARBUILT': '2001',
+        'ceilings': [
+            {
+                'label': 'Main attic',
+                'type_english': 'Attic/gable',
+                'type_french': 'Combles/pignon',
+                'nominal_rsi': '2.864',
+                'effective_rsi': '2.9463',
+                'area': '46.4515',
+                'length': '23.875',
+            }
+        ]
```

Comment:
We should have `sample_input_e` start from `sample_input_d` and `merge` in a dict with the necessary changes.